masschallenge/django-accelerator
# [AC-6156] Startup sub pages available to all team members by permission

### Changes introduced in [AC-6156](https://masschallenge.atlassian.net/browse/AC-6156):
- Update Startup profile subnav item to remove edit profile child item

### How to test
- Checkout to this branch on both accelerate and django-accelerator
- make migrate on impact to apply migrations
- Login as [54761-user@example.com](http://localhost:8181/admin/simpleuser/user/44670/masquerade/). This is a startup admin
- [Go to the the startup dashboard](http://localhost:8181/startup/16890/dashboard/) 
- Notice that the subnav now has a Profile item that goes to view profile and an edit profile button
